src/pages/BooksPage.js

```diff
@@ -0,0 +1,5 @@
+const BooksPage = () => (
+  <h1> Books </h1>
+);
+
```

Comment:
- [ ] The book should have a form and a submit button.

![rrrrrr](https://user-images.githubusercontent.com/61562626/140954369-77316eed-3975-438e-b2e8-5906a09b6e91.jpeg)